templates/chain.html

```diff
@@ -310,6 +310,9 @@ <h1 id="time-tactic" class="member-title"></h1>
                   </div>
                 </li>
               </div>
+              <div>
```

Comment:
good change. let's move this button to the left pane (under or over the delete operation button).